internal/httpserve/route/login.go

```diff
@@ -20,14 +25,16 @@ import (
 // without the user having to log in again. The refresh token overlaps with the access
 // token to provide a seamless authentication experience and the user can refresh their
 // access token so long as the refresh token is valid.
+var logger *zap.Logger
+
 func registerLoginHandler(router *echo.Echo, h *handlers.Handler) (err error) {
 	_, err = router.AddRoute(echo.Route{
 		Method: http.MethodPost,
 		Path:   "/login",
 		Handler: func(c echo.Context) error {
 			return h.LoginHandler(c)
 		},
-		Middlewares: []echo.MiddlewareFunc{middleware.Recover()},
+		Middlewares: []echo.MiddlewareFunc{middleware.Recover(), dump.BodyDump(logger), secure.Secure(), ratelimit.RateLimiter(), cachecontrol.New()},
```

Comment:
```suggestion
		Middlewares: []echo.MiddlewareFunc{middleware.Recover()},
```

See comment above, you should be bale to remove all these from the handler registration